data/meetings.json

```diff
@@ -5,63 +5,58 @@
     "time": "7:15"
   },
   {
-    "name": "Financial",
+    "name": "R&D",
     "day": "MON",
-    "time": "6:00"
+    "time": "7:00"
   },
   {
-    "name": "Evals",
+    "name": "Eboard",
     "day": "MON",
-    "time": "6:00"
+    "time": "8:30"
   },
   {
-    "name": "Imps",
-    "day": "MON",
-    "time": "7:00"
+    "name": "PR",
+    "day": "TUE",
+    "time": "6:30"
   },
   {
     "name": "Upgrade Committee",
     "day": "MON",
     "time": "7:00"
   },
   {
-    "name": "Imagine RIT",
-    "day": "MON",
-    "time": "8:00"
-  },
-  {
-    "name": "PR",
-    "day": "TUE",
-    "time": "7:30"
+    "name": "Evals",
+    "day": "WED",
+    "time": "7:00"
   },
   {
     "name": "History",
-    "day": "TUE",
-    "time": "8:00"
-  },
-  {
-    "name": "History Discussion",
-    "day": "TUE",
-    "time": "8:30"
-  },
-  {
-    "name": "Social",
     "day": "WED",
     "time": "7:30"
   },
   {
-    "name": "Eboard",
+    "name": "History Discussion",
     "day": "WED",
-    "time": "9:00"
+    "time": "8:00"
   },
   {
-    "name": "R&D",
+    "name": "Social",
     "day": "THU",
     "time": "7:00"
   },
   {
     "name": "OpComm",
     "day": "THU",
     "time": "8:00"
+  },
+  {
+    "name": "Eboard",
+    "day": "WED",
+    "time": "9:00"
+  },
+  {
+    "name": "Financial",
+    "day": "TBD",
+    "time": "0:00"
   }
```

Comment:
Hopefully @tflaglor1 can pick this soon, so it might be better to wait for this to be official so we can do 1 re-build instead of 2